### PR TITLE
feat(i18n): add base Italian translations

### PR DESCRIPTION
**Pull request by mithenks**
This PR add Italian translations.

### CODE REVIEW

File: admin/src/components/CMEditView/RightLinksCompo/Summary/PreviewChecks/index.js [hunk 1]
```diff
@@ -26,7 +26,10 @@ const SeoChecker = ({ checks }) => {
   return (
     <Box paddingTop={4}>
       <Typography variant="omega" fontWeight="semiBold">
-        SEO Summary
+	  {formatMessage({
+		  id: getTrad('SEOComponent.title.summary'),
```
Comment (Mcastres) [inline admin/src/components/CMEditView/RightLinksCompo/Summary/PreviewChecks/index.js:30]: You are not creating translations for `SEOComponent.title.summary` in en and fr but a `SEO Summary` one instead.

Can you please remove the `SEO Summary`, add the `SEOComponent.title.summary` in both fr and en?

File: admin/src/components/CMEditView/RightLinksCompo/Summary/BrowserPreview/index.js [hunk 2]
```diff
@@ -43,7 +43,10 @@ const BrowserPreview = ({ modifiedData, setIsVisible }) => {
     >
       <ModalHeader>
         <Typography fontWeight="bold" textColor="neutral800" as="h2" id="title">
-          SEO Plugin
+          {formatMessage({
+            id: getTrad('plugin.name'),
```
Comment (Mcastres) [inline admin/src/components/CMEditView/RightLinksCompo/Summary/BrowserPreview/index.js:47]: Can it be `Plugin.name`?

File: admin/src/translations/en.json [hunk 3]
```diff
@@ -75,5 +76,6 @@
   "Improvements": "Improvements",
   "Bad": "Bad",
   "Social-preview.alert": "Complete you social component to see the preview",
-  "Information": "Information"
+  "Information": "Information",
+  "SEO Summary": "SEO Summary"
```
Comment (Mcastres) [inline admin/src/translations/en.json:80]: Must be replaced. See comment above :)

**Review by Mcastres — CHANGES_REQUESTED** [review]

**Comment by Mcastres** [comment]
Thanks for your translation PR @mithenks 💪

**Comment by ecosse3** [comment]
Any idea when it will be merged? I want to provide Polish translations 😄

**Comment by SalahAdDin** [comment]
@mithenks could you check this?

**Comment by SalahAdDin** [comment]
@mithenks  Why did you close it?

**Pull request closed without merging**